build/metadata/github-actions/README.md

```diff
@@ -74,6 +74,21 @@ If the action fails, there may be a problem with your configuration. To investig
 
 ## Additional Options
 
-Additional configuration options can be found at the bottom of the [LaunchDarkly GitHub Action documentation](https://docs.launchdarkly.com/home/code/github-actions#additional-configuration-options).
+<!-- action-docs-inputs -->
+## Inputs
 
-For information about the underlying LaunchDarkly Code References command-line tool, take a look at [this repository](https://github.com/launchdarkly/ld-find-code-refs).
+| parameter | description | required | default |
+| - | - | - | - |
+| accessToken | A token with write access to the LaunchDarkly project. | `true` |  |
+| allowTags | Enable storing references for tags. The tag will be listed as a branch. | `false` | false |
+| baseUri | The base URL of the LaunchDarkly server for this configuration. | `false` | https://app.launchdarkly.com |
+| contextLines | The number of context lines above and below a code reference for the job to send to LaunchDarkly. By default, the flag finder will not send any context lines to LaunchDarkly. If < 0, no source code will be sent to LaunchDarkly. If 0, only the lines containing flag references will be sent. If > 0, will send that number of context lines above and below the flag reference. A maximum of 5 context lines may be provided. | `false` | 2 |
```

Comment:
```suggestion
| contextLines | The number of context lines above and below a code reference for the job to send to LaunchDarkly. By default, the flag finder will not send any context lines to LaunchDarkly. If < 0, it will send no source code to LaunchDarkly. If 0, it will send only the lines containing flag references. If > 0, it will send that number of context lines above and below the flag reference. You may provide a maximum of 5 context lines. | `false` | 2 |
```